cpp/notebooks/ros1_widget_tutorial.ipynb

```diff
@@ -0,0 +1,310 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9ed828cb-43ae-4592-a06c-6b41275be43e",
+   "metadata": {},
+   "source": [
+    "# **Tutorial to explore ROS 1**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4e853e6e-b262-4634-8419-743b78a0eb2b",
+   "metadata": {},
+   "source": [
+    "## **Prepare the environment for roscpp**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8210c14f-cdbc-45a7-bc90-7f25829883ce",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! python3 ../scripts/generate_cling_3rd_party.py roscpp"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "197ce37a-1dba-4e24-850e-c353053afa42",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#include \"load_roscpp.h\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de94cfa3-162b-4b59-89c4-e1ed915b27ef",
+   "metadata": {},
+   "source": [
+    "### **Let's Begin with ROS Param**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7e106b28-e60f-46c5-8a23-10ef1912319a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#include \"jupyter_ros_utils/ros1_param.hpp\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "35bcbb65-2a54-4244-8651-05b6bc277627",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ros_init(\"param_widget\")"
```

Comment:
Can we have a note above this cell to remind user to run their roscore please?
Otherwise it will stuck here

Reply:
Added.